### DISCUSSION TITLE
[BP#16] Utiliser certains forks applicatifs orientés "performance"

### DISCUSSION THREAD

**Discussion by ldesgrange** [cnumr/best-practices#228] [Bonnes pratiques] [answered]
La BP me semble bonne.

Par contre dans le livre l'exemple MySQL vs Percona affiche un graphique de comparaison de performances fait par la société Percona, donc potentiellement biaisé. De plus il semblerait que MariaDB ait des performances encore meilleures (mais ce n'est pas un fork orienté performances).
Peut-être supprimer cet exemple, le remplacer par Redis vs. KeyDB ?

**Comment by ACTLEM** [comment, marked as answer]
Issue créée pour modification effective: https://github.com/cnumr/best-practices/issues/256

**Comment by ldesgrange** [comment]
En cherchant un peu plus, outre le problème de licence, le fork de MySQL pour faire MariaDB avait aussi comme objectifs les corrections de bugs et améliorations de performances, donc on peut peut-être juste remplacer Percona par MariaDB.

L'exemple d'utiliser Pressflow à la place de Drupal ne me semble plus bon non plus (Pressflow semble être en fin de vie, il s'arrête à Drupal 6/7 (avec très peu de changements pour la 7), la version courante de Drupal étant la 9).

Comme MySQL/MariaDB et Redis/KeyDB sont des bases de données, je cherchais un exemple pour un autre type de logiciel pour remplacer l'exemple Drupal/Pressflow mais je ne trouve pas… Si vous avez des suggestions ?

Sinon, dans certains cas il y a des implémentations différentes orientés performances (au lieu de forks), je ne sais pas si ça compte comme exemple que l'on peut mettre ici, comme par exemple utiliser PyPy au lieu de CPython pour exécuter du code Python ou GraalVM au lieu de HotSpot pour exécuter du Java.
